pkg/kubelet/kubelet.go

```diff
@@ -1409,6 +1427,11 @@ func (kl *Kubelet) Run(updates <-chan kubetypes.PodUpdate) {
 		if utilfeature.DefaultFeatureGate.Enabled(features.NodeLease) {
 			go kl.nodeLeaseController.Run(wait.NeverStop)
 		}
+
+		// start informer for CSIDriver
+		if utilfeature.DefaultFeatureGate.Enabled(features.CSIDriverRegistry) {
+			go kl.informerFactory.Start(wait.NeverStop)
```

Comment:
we should do a `WaitForCacheSync` after this like in the attach detach controller

Reply:
I think we should do this inside volumeManager instead of kubelet, so we don't block kubelet initialization.

Reply:
WaitForCacheSync is called in volumeManager Run right now.

Reply:
Can informerFactory be nil in standalone kubelet?

Reply:
If kubeclient is nil, we skip the block to initialize informers.  So it should be already skipped.